nixos/modules/misc/ids.nix

```diff
@@ -339,6 +339,7 @@
       rss2email = 312;
       cockroachdb = 313;
       zoneminder = 314;
+      coredns = 315;
```

Comment:
Is there need to allocate user ID? Usually we do this for things which have state (like databases) on disk. If thing is ephemeral, then user ID can be ephemeral as well

Reply:
An example: https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/networking/xrdp.nix